fuel-vm/Cargo.toml

```diff
@@ -77,35 +77,35 @@ tokio-rayon = "2.1.0"
 [features]
 default = ["std"]
 std = [
-    "alloc",
-    "fuel-crypto/std",
-    "fuel-types/std",
-    "fuel-asm/std",
-    "fuel-tx/std",
-    "itertools/use_std",
+  "alloc",
+  "fuel-crypto/std",
+  "fuel-types/std",
+  "fuel-asm/std",
+  "fuel-tx/std",
+  "itertools/use_std",
 ]
-alloc = ["fuel-asm/alloc", "fuel-tx/alloc", "fuel-tx/alloc"]
+alloc = ["fuel-asm/alloc", "fuel-tx/alloc", "fuel-crypto/alloc"]
```

Comment:
I don't have enough context to understand this line. Why are we switching the alloc feature?